src/components/molecules/main-nav/main-nav.tsx

```diff
@@ -5,47 +5,50 @@ import {
   DrawerOverlay,
   DrawerContent,
   DrawerCloseButton,
-  useDisclosure
+  useDisclosure,
+  Button
 } from "@chakra-ui/react";
 import { Fragment } from "react";
 import NavLink from "components/atoms/nav-link/nav-link";
 import Icon from "components/atoms/icon/icon";
 import { FiMenu, FiExternalLink } from "react-icons/fi";
 import { useRouter } from "next/router";
-
+import DefaultButton from "../../atoms/button/button";
 interface MainNavProps {
   navList: NavLink[];
+  mobileList: NavLink[];
 }
 
-const MainNav = ({ navList }: MainNavProps) => {
+const MainNav = ({ navList, mobileList }: MainNavProps) => {
   const { isOpen, onOpen, onClose } = useDisclosure();
   const router = useRouter();
 
   return (
     <nav aria-label="Main">
-      <ul className="hidden gap-4 lg:flex">
+      <ul className="lg:flex space-x-8 hidden items-center">
         {navList.map((navItem, index) => (
           <Fragment key={index}>
-            <li className="flex items-center gap-2">
+            <li>
               <NavLink
                 activeLink={router.pathname === navItem.url}
                 {...navItem}
               />
-              {navItem.externalLink && (
-                <Icon
-                  Icon={FiExternalLink}
-                  size={22}
-                  iconAlt="external link for TechIsHiring newsletter"
-                />
-              )}
```

Comment:
```
{navItem.externalLink && (
  <Icon
    Icon={FiExternalLink}
    size={22}
    iconAlt="external link for TechIsHiring newsletter"
 />
)}
```

Please replace this code block. This is there for accessibility.

Reply:
please what do you mean by replacing this code block?

Reply:
You deleted it in your PR. Could you please put it back?